migrations/V187__Add_System_Intake_Doc_Types.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TYPE system_intake_document_type RENAME VALUE 'DRAFT_ICGE' TO 'DRAFT_IGCE';
```

Comment:
do we need to update the column values for this?

Reply:
Interestingly I don't think we do! I've done this before, and since Postgres v10 this functionality has been made pretty simple.

This `ALTER TYPE ... RENAME VALUE` syntax will automatically rename values in columns, arrays, etc. (anywhere that this type is referenced by it's enum type)

We'd only have to be careful if we're referencing this value as `::TEXT` anywhere (and then the psql engine wouldn't find it since it's no longer typed), but I think we should be safe from that as far as I can tell! 😁

Reply:
...what Clay said. From what I read, Postgres can handle this just from the single command.

Reply:
nice! that's really cool - thanks!